maxim/logger/livekit/worker.py

```
@@ -0,0 +1,103 @@
+        scribe().info(f"[Internal][MaximSDK][LiveKit:{self.__class__.__name__}] Worker started")
+    elif action == "worker_stopped":
+        scribe().info(f"[Internal][MaximSDK][LiveKit:{self.__class__.__name__}] Worker stopped")
+    elif action == "worker_error":
+        scribe().error(f"[Internal][MaximSDK][LiveKit:{self.__class__.__name__}] Worker error")
+    elif action == "worker_status_changed":
+        scribe().info(
+            f"[Internal][MaximSDK][LiveKit:{self.__class__.__name__}] Worker status changed"
+        )
+
+
+def intercept_emit(self, *args, **kwargs):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)